rundeckapp/grails-app/controllers/rundeck/controllers/FrameworkController.groovy

```diff
@@ -3500,5 +3509,14 @@ class FrameworkController extends ControllerBase implements ApplicationContextAw
         }
         render(status: HttpServletResponse.SC_NO_CONTENT)
     }
+
+    private def nodeSourceFileExceededMaxSize(ByteArrayInputStream bais){
```

Comment:
since this is a new method, please use @CompileStatic annotation

Reply:
Done, due @CompileStatic, had to change the bais.count to bais.available() to avoid conflicts